src/test/java/actions/BaseRestActions.java

```diff
@@ -0,0 +1,32 @@
+package actions;
+
+import com.endava.models.CreatePersonRequest;
+import com.endava.models.CreatePersonResponse;
+import io.restassured.RestAssured;
+
+import static io.restassured.RestAssured.given;
+
+public class BaseRestActions {
+    public static void setup() {
+        RestAssured.baseURI = "https://reqres.in/api";
+    }
+
+    public CreatePersonResponse createPerson(String name, String job) {
+        CreatePersonRequest personRequest = new CreatePersonRequest();
+        personRequest.setName(name);
+        personRequest.setJob(job);
+
+
+        CreatePersonResponse response = given()
+                .header("Content-type", "application/json")
```

Comment:
Good idea to add log().all() to be able to capture request/response in execution log.